.env.dev

```diff
@@ -15,7 +16,7 @@ TWILIO_AUTH_TOKEN=
 TWILIO_NUMBER=
 
 DJANGO_SETTINGS_MODULE=settings.dev
-SECRET_KEY=jkashdkjashdkjh
```

Comment:
ran across a few errors at some points complaining about the secret being < 30 characters